docs/application/native/guides/text-input/tts.md

```diff
@@ -55,15 +55,15 @@ You can set the following parameters about the TTS:
   The private data is a setting parameter for applying keys provided by the TTS engine. Using the `tts_set_private_data()` function, you can set the private data as the corresponding key of the TTS engine.
 
 <a name="info_tts"></a>
-## TTS Information Retrieval
+## TTS information retrieval
 
 You can get the following information about the TTS:
 
 - [Get the current state](#get) of the TTS. The state is also applied as a precondition for each function.
 - Get the default voice.
   - In the TTS, the voice is defined as a combination of the language and the type, such as male or female.
   - You can request the synthesis of the text with your own voice option by the parameter of the add text function. However, if you do not set a specific option, the TTS synthesizes the text with the default voice.
```

Comment:
Line 65: Should we use code tags for "add text" here to specify the function name?

Reply:
Yes, I agree your opnion.
It could be better to find out the detail of function. And to maintain consistency dotnet guide, I will use function name itself here.
Thank you.